pkgs/development/python-modules/symengine/default.nix

```
@@ -1,18 +1,20 @@
-{ lib
-, buildPythonPackage
-, fetchFromGitHub
-, cython
-, cmake
-, symengine
-, pytest
-, sympy
-, python
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  cython,
+  cmake,
+  symengine,
+  pytest,
+  sympy,
+  python,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "symengine";
   version = "0.11.0";
-  format = "setuptools";
+  build-system = [ setuptools ];
```

```suggestion
  pyproject = true;

  build-system = [ setuptools ];
```
Please add `pyproject = true` to avoid the legacy build.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

But it isn't a pyproject...?

---

The document says,
> Note that the pyproject format falls back to using setuptools, so you can use pyproject = true even if the package only has a setup.py. 

> When unset, the legacy setuptools hooks are used for backwards compatibility.